### ISSUE TITLE
The image could not be loaded and the password was encrypted

### ISSUE THREAD

**Issue by lmh122** [trustedsec/social-engineer-toolkit#768]
Description:	Kali GNU/Linux Rolling
Release:	2020.2
Codename:	kali-rolling
![2020-06-22 18-11-35屏幕截图](https://user-images.githubusercontent.com/57933165/85276146-eefe2480-b4b3-11ea-95ee-beafe1cf5a71.png)

The image could not be loaded and the password was encrypted

**Comment by stapleton159** [comment]
Similar issue on another thread although the user did not specify password encryption. Certain websites use a private key to decode encrypted passwords through security algorithms. You can see this because your password output is a 40 hexadecimal character line. **You could download the source into a index.html file and then use setoolkit to take out this variable.**

 If you inspect this website and bring up network while entering the information:
http://218.196.240.68/eams/static/scripts/sha1.js
The last line in this script is 
**e.SHA1=j._createHelper(m);e.HmacSHA1=j._createHmacHelper(m)**

This makes me think that the password is encoded through the Crypto JS library with SHA-1 using also HMAC authentication hashing, it is one way encryption hashing, meaning you can't decrypt it.

MORE INFO:
https://stackoverflow.com/questions/2434154/how-to-decrypt-a-string-encrypted-with-hmacsha1

So in short it is not encrypted, but hashed once and then hashed again with the party with the private key, it is more of a website _issue_ rather than setoolkit. You would have to break the hashing algorithm.

There are certain websites... to avoid cloning especially for training, try to download from source.

**Comment by HackingDave** [comment]
Closing as an unauthorized account/password or account-access request. SET is for authorized security testing only and this repository will not provide help for unauthorized access.
